helpers/api_helper/json_api_response.go

```diff
@@ -18,13 +20,25 @@ func (e *ErrorResponseObject) NewErrorResponse() gin.H {
 }
 
 type DataResponseObject struct {
-	ID         string      `json:"id"`
-	Type       string      `json:"type"`
-	Attributes interface{} `json:"attributes"`
+	ID            string      `json:"id"`
+	Type          string      `json:"type"`
+	Attributes    interface{} `json:"attributes,omitempty"`
+	Relationships interface{} `json:"relationships,omitempty"`
```

Comment:
It looks like `omitempty` does mothing here in the test `TestGetRelationshipsWithValidDataResponseObject`, both `Attributes` and `Relationships` are returned (which is good by the way). I think `omitempty` could be removed entirely. 

I found [this article](https://www.sohamkamani.com/golang/omitempty/) which explains the usage of `omitempty`

Reply:
Fixed in 7960a60637d858217f33899385e892a0abd5eae6